core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsAddress.java

```diff
@@ -77,7 +77,7 @@ public void doWriteObject(ObjectOutput output, JGroupsAddress address) throws IO
       public JGroupsAddress doReadObject(ObjectInput unmarshaller) throws IOException, ClassNotFoundException {
          try {
             org.jgroups.Address address = org.jgroups.util.Util.readAddress(unmarshaller);
-            return (JGroupsAddress) JGroupsAddressCache.fromJGroupsAddress(address);
+            return (JGroupsAddress) JGroupsAddressCache.fromJGroupsAddressUnchecked(address);
```

Comment:
1. Because in master the signature of the method is `org.infinispan.remoting.transport.Address fromJGroupsAddress(Object address)`.
2. Because even local caches can have stores.